### DISCUSSION TITLE
Using discovery API with Nodes

### DISCUSSION THREAD

**Discussion by cjyar** [kube-rs/kube#639] [Q&A] [answered]
I'm trying to use the discovery API with `Node` objects, but calling `pinned_kind` returns `Error: Api(ErrorResponse { status: "404 Not Found", message: "\"404 page not found\\n\"", reason: "Failed to parse error data", code: 404 })`.

```rust
use kube::{Client, Error, api::GroupVersionKind, discovery};

#[tokio::main]
async fn main() -> Result<(), Error> {
    let client = Client::try_default().await?;

    let gvk = GroupVersionKind{
        group: "core".into(),
        version: "v1".into(),
        kind: "Node".into(),
    };
    let _disco = discovery::pinned_kind(&client, &gvk).await?;

    Ok(())
}
```

Am I doing something wrong? This works with other resource types.

**Comment by clux** [comment, marked as answer]
I believe the `group` keys should just be `""` there. That's what the group is actually called.

**Comment by cjyar** [comment]
You're quite right. Thanks for the quick response!
